main.go

```diff
@@ -298,7 +347,11 @@ func collector(pid int, fh *os.File) error {
 
 	for _, proc := range processes {
 		// TODO Combine these lines
-		procstats := collectStatsFor(proc)
+		procstats, err := collectStatsFor(proc, available)
+		if err != nil {
+			// Process probably ended between findAllProcesses and here.
+			continue
```

Comment:
Nice! I actually caught this case earlier while doing timed-out `stress` tests locally. The process died before I sent the signal to the script.